src/utils/telemetryHelper.ts

```diff
@@ -0,0 +1,29 @@
+const TELEMETRY_CHAT_PATH = "/v1/telemetry-chat";
+
+export function sendTelemetryEvent({
```

Comment:
move to the /src/services/refact directory.

There'll you'll find similar code for communicating with the lsp and getting info from the state :)

Then that way the telemetry could be use as a middle where like in `src/app/middleware` this will let the service be called as a side effect when specific actions happen :)